Server/tests/integration/test_manage_ui.py

```
@@ -0,0 +1,428 @@
+    font-size: 24px;
+    color: white;
+}"""
+
+
+class TestManageUIPathValidation:
+    """Tests for path validation logic."""
+
+    def test_create_rejects_path_outside_assets(self, monkeypatch):
+        captured = {}
+
+        async def fake_send(ctx, instance, cmd, params, **kwargs):
```

Ruff (0.15.2)

[warning] Unused function argument: `ctx`

(ARG001)